src/briefcase/commands/new.py

```diff
@@ -516,16 +645,89 @@ def verify_tools(self):
+            merged_pyproject = "\n\n\n".join((new_pyproject, pyproject))
+        else:
+            merged_pyproject = new_pyproject
+
+        with open(pyproject_file, "w") as file:
+            file.write(merged_pyproject)
+
     def __call__(
         self,
         template: Optional[str] = None,
         template_branch: Optional[str] = None,
+        target: Optional[Path] = None,
         **options,
     ):
         # Confirm host compatibility, and that all required tools are available.
         # There are no apps, so finalize() will be a no op on app configurations.
-        self.finalize()
 
-        return self.new_app(
-            template=template, template_branch=template_branch, **options
-        )
+        self.finalize()
+        if not target:
+            return self.new_app(
+                template=template, template_branch=template_branch, **options
+            )
+        elif not target.parent.exists():
+            raise ValueError("The parent directory of ``--target`` must exist.")
+        elif (target / "pyproject.toml").exists():
+            with open(target / "pyproject.toml", "rb") as file:
+                pyproject = tomllib.load(file)
+            if "tool" in pyproject and "briefcase" in pyproject["tool"]:
+                raise ValueError(
+                    f"[tool.briefcase] already in {target / 'pyproject.toml'}."
+                    " Cannot initialise briefcase for an application that already is packaged with briefcase."
+                )
+
+        with TemporaryDirectory() as output_path:
+            output_path = Path(output_path)
+            base_path = self.base_path
+            self.base_path = output_path
+            try:
+                app_name = self.new_app(
+                    template=template,
+                    template_branch=template_branch,
+                    target=target,
+                    **options,
+                )
+            finally:
+                self.base_path = base_path
+            project_dir = output_path / app_name
+
+            if target.exists():
+                self.merge_or_copy_pyproject(project_dir, target)
+                license_file = target / "LICENSE"
+                if not license_file.is_file():
+                    self.logger.warning(
+                        f"License file not found in {target}. You should either create a new license file in {target}"
+                        " or rename an already existing license file to LICENSE."
+                    )
+
+                changelog_file = target / "CHANGELOG"
+                if not changelog_file.is_file():
+                    self.logger.warning(
+                        f"Changelog file not found in {target}. You should either create a new changelog file in"
+                        f" {target} or rename an already existing changelog file to CHANGELOG."
+                    )
+            else:
+                shutil.copytree(project_dir, target)
```

Comment:
Agreed! We have attempted to refactor it out into sub methods now. While we didn't change the tests, the test coverage is still 100%. Is this OK, or should we also add tests explicitly for these new utility methods before merging?

Reply:
If coverage is still 100%, there's no need to add tests. If there's a new edge case introduced by potential arguments, it might be worth adding some additional explicit tests; but otherwise, I'm comfortable with the logic being tested implicitly by usage higher up.